repo/packages/J/jenkins/10/config.json

```diff
@@ -63,6 +63,22 @@
                 }
             }
         },
+         "roles": {
+              "description": "Role configuration properties for Jenkins on DC/OS.",
+              "type": "object",
+              "properties": {
+                "jenkins-master-role": {
+                    "description": "The accepted resource roles (e.g. slave_public) that the Jenkins master itself runs as using Marathon. By default, this will deploy to any agents with the * role.",
```

Comment:
I would change or remove the example, unless we want users to run masters on nodes with `slave_public`. I could see this example being interpreted as a suggested default.